.github/labels.md

```diff
@@ -182,10 +182,23 @@ with `@`-mentions if someones or some groups special knowledge is required.
 
 #### Label `status:invalid`
 
-The Issue contains content that fails to address its intention.  An example
-would be a feature request wasn't discussed before in the forums or for Pull
-Requests those requests that compete to address the same topic, but were not
-chosen to be merged.  Those Issues shouldn't be merged.
+The Issue doesn't classified as formal valid Issue report or applicable Pull
+Request.  Reasons for this can be manifold:
+
+* The Issue isn't describing a bug but intended behaviour.
+* The Issue opposes the design goals of the game.
+* The issue contains SPAM.
+* The description is too sparse to make sense out of the intent of the Issue
+  and the Reporter is unresponsive or unwilling to give further details.
+* The Feature implemented wasn't discussed previously with the team.
+* The Pull Request is a merge salad à la Chef and the developer is unable or
+  unwilling to fix this.
+* The commits contain intentional written game-braking or offending changes and
+  the developer is unwilling to change those.
+* Multiple Pull Requests compete to address the same topic, but were not chosen
```

Comment:
I updated the `status:duplicate` to catch this case and removed this list entry.

(Hopefully) fixed.